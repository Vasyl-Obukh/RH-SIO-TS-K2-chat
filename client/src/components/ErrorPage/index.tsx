import * as React from 'react';

function ErrorPage() {
  return <h1>You should not be here...</h1>;
}

export default ErrorPage;