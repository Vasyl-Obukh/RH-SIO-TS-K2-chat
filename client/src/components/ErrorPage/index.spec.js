import React from 'react';
import { shallow } from 'enzyme';
import ErrorPage from './index';

describe('<ErrorPage> tests', () => {
  test('Should render correctly', () => {
    const component = shallow(< ErrorPage / >);
    expect(component).toMatchSnapshot();
  });
});