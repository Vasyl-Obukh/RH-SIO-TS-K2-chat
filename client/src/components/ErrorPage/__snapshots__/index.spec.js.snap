// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage> tests Should render correctly 1`] = `
<h1>
  You should not be here...
</h1>
`;
